### PR TITLE
Print ccache/clcache hit rate

### PR DESCRIPTION
**Pull request by Avin0323**
### PR types

Others

### PR changes

Others

### Describe

打印PR在编译时的ccache/clcache命中率数据，通过`CACHE_HIT_STATISTICS`开关进行管理。
image
image

### CODE REVIEW

File: paddle/scripts/paddle_build.bat [hunk 1]
```diff
@@ -268,14 +268,21 @@ echo Build third_party successfully!
 
 set build_times=1
 :build_paddle
+:: reset clcache zero stats for collect PR's actual hit rate
+if "%CACHE_HIT_STATISTICS%"=="ON" clcache.exe -z
```
Comment (zhwesky2010) [inline paddle/scripts/paddle_build.bat]: I think CACHE_HIT_STATISTICS is not need, and it is WITH_CLCACHE, when WITH_CLCACHE=ON, collect_clcache_hits; WITH_CLCACHE=OFF: not collect_clcache_hits, or echo "clcache hit rate: 0%"

Comment (Avin0323) [reply paddle/scripts/paddle_build.bat]: It maybe fail when `clcache` command not exsits.

Comment (zhwesky2010) [reply paddle/scripts/paddle_build.bat]: can ``clcache --v`` adjuge first

Comment (Avin0323) [reply paddle/scripts/paddle_build.bat]: It is not necessary, `clcache` installed at first in script.

**Review by zhwesky2010 — APPROVED** [review]

**Pull request merged**